### PR TITLE
fix: Added tests Board component

### PR DESCRIPTION
**Pull request by hyperxtend**

### CODE REVIEW

File: src/components/board/component.test.jsx [hunk 1]
```diff
@@ -4,11 +4,52 @@ import { mount } from 'enzyme';
...
+  const wrapper = mount(<Board squares={squares} onClick={onClick} />);
+
+  it('onClick event works on 1st button', () => {
+    wrapper.find('button.square').at(0).simulate('click');
+    expect(onClick).toBeCalledWith(0);
+  });
+
+  it('onClick event works on 2nd button', () => {
+    wrapper.find('button.square').at(1).simulate('click');
+    expect(onClick).toBeCalledWith(0);
```
Comment (strange-developer) [inline src/components/board/component.test.jsx]: Shouldn't `onClick` be called with `1`? Same for tests in this file

**Review by strange-developer — CHANGES_REQUESTED** [review]

File: src/components/board/component.test.jsx (hunk 1 above)
Comment (hyperxtend) [reply src/components/board/component.test.jsx]: Yes, I should actually

File: package.json [hunk 2]
```diff
@@ -4,10 +4,10 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "start": "react-scripts start --NODE_ENV=production && node index.jsx",
+    "start": "react-scripts start --NODE_ENV=production&&node index.jsx",
```
Comment (strange-developer) [inline package.json]: Are the spaces not required on the ampersands? It should be AFAIK

**Review by strange-developer — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (hyperxtend) [reply package.json]: Yes they are

**Review by strange-developer — APPROVED** [review]

**Pull request merged**